### PR TITLE
Final layout

### PR DESCRIPTION
**Pull request by harrycoulton**
lots of changes and an overhaul to the design

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,33 +8,267 @@
     <title>index</title>
     <link rel="stylesheet" type="text/css" href="styles.css">
     <link href="https://fonts.googleapis.com/css?family=Montserrat&display=swap" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/c4ef94e39c.js"></script>
+
```
Comment (mporam) [inline index.html]: dont need this extra whitespace

File: index.html [hunk 2]
```diff
@@ -8,33 +8,267 @@
...
+
+
 </head>
 
 <body>
     <div class="main-image-container">
         <div class="navbar">
```
Comment (mporam) [inline index.html]: you could use a `<nav>` element here instead, more semantic

File: index.html [hunk 3]
```diff
@@ -8,33 +8,267 @@
...
+                        <i class="proficiency-icon fab fa-js-square"></i>
+                        <i class="proficiency-icon fab fa-react"></i>
+                        <i class="proficiency-icon fab fa-angular"></i>
+                    </div>
+                </div>
+            </div>
+            <div class="parallax-banner">Backend</div>
+            <div class="about-me-container-backend">
+                <div class="about-me-backend-container proficiency-container">
+
```
Comment (mporam) [inline index.html]: why empty line?

File: index.html [hunk 4]
```diff
@@ -8,33 +8,267 @@
...
+                    <i class="proficiency-icon fab fa-php"></i>
+                    <i class="proficiency-icon fab fa-python"></i>
+                    <i class="proficiency-icon fab fa-node"></i>
+                </div>
+            </div>
+        </div>
+        <div class="navbar-bumper" id="portfolio">
```
Comment (mporam) [inline index.html]: why empty div?

File: index.html [hunk 5]
```diff
@@ -8,33 +8,267 @@
...
+        </div>
+        <div class="parallax-banner"> Portfolio</div>
+
```
Comment (mporam) [inline index.html]: more empty lines?

File: index.html [hunk 6]
```diff
@@ -8,33 +8,267 @@
...
+        <div class="portfolio-wrapper">
+
+            <div class="portfolio-main-container">
+                <div class="portfolio-item-row">
+
+                    <div class="portfolio-item portfolio-item-1">
+                        <img src="image-assets/forest-3742949.jpg">
+
+                        <div class="middle">
```
Comment (mporam) [inline index.html]: empty div?

File: index.html [hunk 7]
```diff
@@ -8,33 +8,267 @@
...
+                        </div>
+                        <div class="text-overlay">
+                            <div class="portfolio-image-hover-text">
+                                Project 1: This project does some cool stuff
+                            </div>
+                        </div>
+                    </div>
+                    <div class="portfolio-item portfolio-item-1">
+                        <img src="image-assets/General_Sherman_tree_looking_up.jpg">
+                        <div class="middle">
```
Comment (mporam) [inline index.html]: lots of empty divs?

File: index.html [hunk 8]
```diff
@@ -8,33 +8,267 @@
...
+                        <div class="text-overlay">
+                            <div class="portfolio-image-hover-text">
+                                Project 1: This project does some cool stuff
+                            </div>
+                        </div>
+                    </div>
+
+                </div>
+
+
```
Comment (mporam) [inline index.html]: WHITESPACE?

File: styles.css [hunk 9]
```diff
@@ -1,56 +1,687 @@
...
-}
-
-.navbar a:hover {
-    color: #EDBCED;
-}
-
-
-/* Home area */
\ No newline at end of file
+            /*! normalize.css v2.1.0 | MIT License | git.io/normalize */
```
Comment (mporam) [inline styles.css]: please put normalize in a seperate css file and link to it in your HTML

File: styles.css [hunk 10]
```diff
@@ -1,56 +1,687 @@
...
+                /* 2 */
+            }
+            /* ==========================================================================
+   Tables
+   ========================================================================== */
+            /**
+ * Remove most spacing between table cells.
+ */
+            
+            table {
```
Comment (mporam) [inline styles.css]: why so much indentation?

File: styles.css [hunk 11]
```diff
@@ -1,56 +1,687 @@
...
+            .hexagon {
+                fill: #D2D8AB;
+            }
+            
+            .icon {
+                font-size: 1.5rem;
+            }
+            
+            .icon:visited {
+                color: white;
```
Comment (mporam) [inline styles.css]: please use hex colors

File: styles.css [hunk 12]
```diff
@@ -1,56 +1,687 @@
...
+            }
+            
+            .navbar a,
+            p {
+                color: #ECFAEC;
+                font-size: 1.5rem;
+            }
+            
+            .navbar a:active {
+                color: pink;
```
Comment (mporam) [inline styles.css]: hex

File: styles.css [hunk 13]
```diff
@@ -1,56 +1,687 @@
...
+                flex-direction: column;
+                align-items: center;
+                font-size: 2rem;
+                color: #ECFAEC;
+            }
+            
+            .navbar-bumper {
+                height: 15vh;
+                width: 100%;
+                background: transparent;
```
Comment (mporam) [inline styles.css]: shouldnt need this, its default

File: styles.css [hunk 14]
```diff
@@ -1,56 +1,687 @@
...
+            
+            .about-me-container-backend {
+                background-image: url("image-assets/sunset-1757593.jpg");
+                background-attachment: fixed;
+                background-position: center;
+                background-repeat: no-repeat;
+                background-size: cover;
+                display: flex;
+                justify-content: center;
+                border: 1px solid red;
```
Comment (mporam) [inline styles.css]: hex

File: styles.css [hunk 15]
```diff
@@ -1,56 +1,687 @@
...
+            
+            .proficiency-icon {
+                font-size: 7rem;
+                color: white;
+                padding: 10px;
+            }
+            /* Portfolio area */
+            
+            .portfolio-wrapper {
+                width: 100%;
```
Comment (mporam) [inline styles.css]: think this does nothing?

File: styles.css [hunk 16]
```diff
@@ -1,56 +1,687 @@
...
+                align-items: center;
+                width: 100%;
+            }
+            
+            .portfolio-item-row {
+                display: flex;
+                justify-content: space-around;
+                height: 350px;
+                width: 90%;
+                margin: 2% 0;
```
Comment (mporam) [inline styles.css]: unusual to use %s as margin or padding. sure this is correct?

**Review by mporam — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (harrycoulton) [reply index.html]: done

File: index.html (hunk 2 above)
Comment (harrycoulton) [reply index.html]: done

File: index.html (hunk 3 above)
Comment (harrycoulton) [reply index.html]: done

File: styles.css (hunk 10 above)
Comment (harrycoulton) [reply styles.css]: done

File: index.html [hunk 17]
```diff
@@ -7,34 +7,173 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>index</title>
     <link rel="stylesheet" type="text/css" href="styles.css">
+    <link rel="stylesheet" type="text/css" href="normalize.css">
```
Comment (ashleycoles) [inline index.html:9]: This should be loaded before styles.css, so just swap them around.

File: index.html [hunk 18]
```diff
@@ -7,34 +7,173 @@
...
+        <div class="portfolio-item-row">
+            <div class="portfolio-item portfolio-item-1">
+                <img src="image-assets/kazuend-19SC2oaVZW0-unsplash.jpg">
+                <div class="text-overlay">
+                    <div class="portfolio-image-hover-text">
+                        Project 1: This project does some cool stuff
+                    </div>
+                </div>
+            </div>
+            <div class="portfolio-item portfolio-item-1">
```
Comment (ashleycoles) [inline index.html]: I'm guessing this should be portfolio-item-2 not portfolio-item-1?

File: index.html [hunk 19]
```diff
@@ -7,34 +7,173 @@
...
```
Comment (ashleycoles) [inline index.html]: Same here, should this be portfolio-item-2?

File: index.html [hunk 20]
```diff
@@ -7,34 +7,173 @@
...
```
Comment (ashleycoles) [inline index.html]: And again here

**Review by ashleycoles — CHANGES_REQUESTED** [review]

**Review by ashleycoles — APPROVED** [review]

**Pull request merged**